modules/features/endofyear/src/main/java/au/com/shiftyjelly/pocketcasts/endofyear/views/stories/StoryTopFivePodcastsView.kt

```diff
@@ -1,42 +1,163 @@
+import au.com.shiftyjelly.pocketcasts.compose.components.TextH20
 import au.com.shiftyjelly.pocketcasts.compose.components.TextH30
+import au.com.shiftyjelly.pocketcasts.compose.components.TextH70
+import au.com.shiftyjelly.pocketcasts.compose.preview.ThemePreviewParameterProvider
+import au.com.shiftyjelly.pocketcasts.endofyear.components.PodcastLogoWhite
+import au.com.shiftyjelly.pocketcasts.endofyear.utils.podcastDynamicBackground
+import au.com.shiftyjelly.pocketcasts.localization.R
+import au.com.shiftyjelly.pocketcasts.models.entity.Podcast
 import au.com.shiftyjelly.pocketcasts.repositories.endofyear.stories.StoryTopFivePodcasts
+import au.com.shiftyjelly.pocketcasts.ui.theme.Theme
 
 @Composable
 fun StoryTopFivePodcastsView(
     story: StoryTopFivePodcasts,
     modifier: Modifier = Modifier,
 ) {
     Column(
-        verticalArrangement = Arrangement.Center,
-        modifier = modifier.verticalScroll(rememberScrollState()),
+        horizontalAlignment = Alignment.CenterHorizontally,
+        modifier = modifier
+            .fillMaxSize()
+            .podcastDynamicBackground(story.topPodcasts[0].toPodcast())
+            .verticalScroll(rememberScrollState())
     ) {
-        TextH30(
-            text = "Your Top Podcasts",
-            textAlign = TextAlign.Center,
-            color = story.tintColor,
-            modifier = modifier
-                .fillMaxWidth()
-                .padding(vertical = 16.dp)
+        Spacer(modifier = modifier.height(40.dp))
+
+        Spacer(modifier = modifier.weight(1f))
+
+        Title(story)
+
+        Spacer(modifier = modifier.weight(0.5f))
+
+        PodcastList(story)
+
+        Spacer(modifier = modifier.weight(1f))
+
+        PodcastLogoWhite()
+
+        Spacer(modifier = modifier.height(40.dp))
+    }
+}
+
+@Composable
+private fun Title(
+    story: StoryTopFivePodcasts,
+    modifier: Modifier = Modifier,
+) {
+    val text = stringResource(R.string.end_of_year_story_top_podcasts)
+    TextH30(
+        text = text,
+        textAlign = TextAlign.Center,
+        color = story.tintColor,
+        modifier = modifier
+            .padding(horizontal = 40.dp)
+            .fillMaxWidth()
+    )
+}
+
+@Composable
+private fun PodcastList(story: StoryTopFivePodcasts) {
+    story.topPodcasts.forEachIndexed { index, topPodcast ->
+        PodcastItem(
+            podcast = topPodcast.toPodcast(),
+            position = index,
+            tintColor = story.tintColor
+        )
+    }
+}
+
+@Composable
+fun PodcastItem(
+    podcast: Podcast,
+    position: Int,
+    tintColor: Color,
+    modifier: Modifier = Modifier,
+) {
+    Row(
+        verticalAlignment = Alignment.CenterVertically,
+        modifier = modifier
+            .fillMaxWidth()
+            .padding(horizontal = 40.dp)
+    ) {
+        TextH20(
+            text = "${position + 1}.",
+            color = tintColor,
+            modifier = modifier.padding(end = 14.dp)
         )
```

Comment:
```suggestion
        TextH30(
            text = "${position + 1}.",
            color = tintColor,
            fontWeight = FontWeight.Normal,
            modifier = modifier.padding(end = 14.dp)
        )
```
This might match the design closer.